.github/stale.yml

```diff
@@ -0,0 +1,22 @@
+---
+# copied from https://github.com/curl/curl/blob/master/.github/stale.yml
+# thanks @bagder
+
+# Number of days of inactivity before an issue becomes stale
```

Comment:
I'm not a fan of auto-closing issues, so can this be removed?

Reply:
Sure!

Reply:
Funnily enough it doesn't actually do anything without an action that uses it, so I've removed it 😄